config.default

```diff
@@ -132,6 +132,15 @@ mpasInterpolationMethod = bilinear
 startYear = 1
 endYear = 9999
 
+[index]
+## options related to producing nino index.
+
+# start and end years for the nino 3.4 analysis.  Using out-of-bounds values
+#   like start_year = 1 and end_year = 9999 will be clipped to the valid range
+#   of years, and is a good way of insuring that all values are used.
+startYear = 2
+endYear = 6
```

Comment:
I still very confused about what you want me to change.  For example in configs/edison, 
```
config.20161006bugfix.alpha8.A_WCYCL1850S.ne30_oEC_ICG.edison
```
Is missing most of the options in config.default.  Are you expecting me to add everything in there while respecting run directories and observational paths?